test/features/tests.js

```diff
@@ -100,4 +102,33 @@ describe('service-runner tests', () => {
         })
         .finally(() => process.removeListener('warning', warningListener));
     });
+
+    it('Must increment hitcount metrics when hit', () => {
+        const server = new TestServer(`${__dirname}/../utils/simple_config_one_worker.yaml`);
+        return server.start()
+            .then(() => {
+                HTTP.get('http://127.0.0.1:12345', (res) => {
+                    assert.strictEqual(res.statusCode, 200, 'Must get 200 response');
+                });
+            })
+            .delay(1000)
+            .then(() => {
+                HTTP.get('http://127.0.0.1:9000', (res) => {
```

Comment:
We're generally using the library `preq` for making HTTP requests. You can require it as a devDependency and use it. This would allot to simplify the test as it you wouldn't need to parse the response body manually